content/en/tracing/version_tracking/_index.md

```diff
@@ -14,6 +14,75 @@ further_reading:
 
 This page describes specific ways you can use the `version` tag to better monitor deployments and service behavior. See the [Unified Service Tagging][1] docs for instructions for setting up the tags.
 
+## Using Version Tags on the Service Page
+
+{{< img src="tracing/version_tracking/ServicePage.png" alt="Versions on the Service Page"  style="width:100%;">}}
+
+On the Service page, if the `version` tag is available, the requests widget can be scoped to either of:
+
+- Total Requests by Version
+- Requests per second by Version
+
+The errors widget can be scoped to one of three options that involve the `version` tag:
+
+- Total Errors by Version
+- Errors per second by Version
+- % Error rate by Version
+
+All of these can be exported to dashboards and monitors.
+
+## Version Summary Table
+
+Services configured with version tags will have a `version` section on the Service page, immediately below the main service health graphs.  This table will show all versions of the service that were active during the selected time interval, with currently active services at the top.
+
+By default you will see:
+
+- The version name(s) deployed for this service over the timeframe.
+- The times traces corresponding to this version was first and last seen.
+- An indication of how many types of errors appear in each version that did not appear in the immediately previous version.  Note that this is a not a guarantee the version introduced these errors, but an indication that this error was not seen for the previous version.  Based on how you are performing deployments, it can be a great way to begin investigating errors.
+- Requests per second.
+- Error rate as a percentage of total requests.
+
+Additionally, fields can be added or removed from this overview table and your selections will be saved.  The additional fields are:
+
+- Endpoints that appear in a version that did not appear in the previous version.
+- Time active, showing the length of time from the first trace to the last trace sent to Datadog for that version.
+- Total number of Requests.
+- Total number of Errors.
+- Latency measured by p50, p75, p90, p95, p99, or the true max.
+
+{{< img src="tracing/version_tracking/VersionComparison.png" alt="Versions on the Service Page"  style="width:100%;">}}
+
+**Note:** The `version` section of of the service page will only appear if there is more than one version reporting in the selected time interval at the top of the page.
+
+## Version Comparison
+
+Clicking on any version row in the Version Summary table will open a detailed version comparison page, allowing you to compare two specific version of the same service.  By default, the comparison will compare the version clicked to the immediately previous version, but this can be changed to compare any two versions within the past 30 days.
+
```

Comment:
_By default, the comparison will compare the version clicked to the immediately previous version, but this can be changed to compare any two versions within the past 30 days._

seems a little awkward, what about something like:

_By default, the selected version will be compared to the immediately previous version but this can be changed to compare any two versions within the past 30 days._